atlas-db-controller/controller.go

```diff
@@ -273,6 +376,7 @@ func (c *Controller) syncDatabase(key string) error {
 	var p plugin.DatabasePlugin
 	var s *atlas.DatabaseServer
 	if db.Spec.Server != "" {
+		// XXX this implies the same namespace for database and server. Do we want this?
```

Comment:
Mark it as a TODO. For now everything is same namespace but we may want to change that.